torch/_tensor_docs.py

```diff
@@ -4238,6 +4238,18 @@ def callable(a, b) -> number
 
 """)
 
+add_docstr_all('to_sparse_bsr',
+               r"""
+to_sparse_bsr(blocksize) -> Tensor
+Convert a CSR tensor to a block sparse row (BSR) storage format of given blocksize.
+Example::
```

Comment:
I'm worried that it might not render correctly without blank lines in between the signature, description and example as in other docstrings.
[There's a step of uploading docs for preview ](https://github.com/pytorch/pytorch/runs/6416590001?check_suite_focus=true) but I can't find the actual link to that. Previously bot was posting it to the summary comment https://github.com/pytorch/pytorch/pull/77366#issuecomment-1125358927.